### ISSUE TITLE
Q for D

### ISSUE THREAD

**Issue by knif-knif** [MrChenFeng/SSR_BMVC2022#7]
utils/funcs.py line 30 
```
return - F.cosine_similarity(p, z.detach(), dim=-1).mean()
```
It's negative， Should it be 
```
return 1 - F.cosine_similarity(p, z.detach(), dim=-1).mean()
```

**Comment by MrChenFeng** [comment]
hi, the constant is omitted as it won't affect the gradient.
